app/domain/authentication/strategy.rb

```diff
@@ -77,6 +79,24 @@ def conjur_token(input)
       raise e
     end
 
+    def login(input)
+      authenticator = authenticators[input.authenticator_name]
+
+      validate_authenticator_exists(input, authenticator)
+      validate_security(input)
+
+      key = authenticator.login(input)      
+      raise InvalidCredentials unless key
+
+      # TODO: Should be audit_login_success
```

Comment:
What's this `TODO` for? Something worth fixing in this branch?

Reply:
No, but I will clean up the todo comments.